core/templates/pages/library-page/search-bar/search-bar.component.spec.ts

```diff
@@ -224,7 +224,7 @@ describe('Search bar component', () => {
         value: 'search'
       }
     };
-    expect(component.searchToBeExec(search)).toBeNull();
+    expect(component.searchToBeExec(search)).toBeUndefined();
```

Comment:
Note:- This has been updated `.toBeUndefined()` because earlier `searchToBeExec` was returning a null value but return type for that particular function is `void` therefore usage of null was removed. So `component.searchToBeExec(search)` would be undefined and not null